migrations/001_create_tables.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE User (
+    ID int NOT NULL,
```

Comment:
Why ID is uppercase, but other fields lowercase?
Please make all fields lowercase. (id)
[check](https://dev.mysql.com/doc/refman/8.0/en/identifier-case-sensitivity.html#:~:text=Table%20names%20are%20stored%20in,database%20names%20and%20table%20aliases.)